ServiceWorkerRegistration/ServiceWorkerRegistration_index_property.html

```diff
@@ -0,0 +1,72 @@
+<div>{{draft}}{{DefaultAPISidebar("Service Worker API")}}</div>
+
+<p class="summary">The <strong><code>index</code></strong> read-only property of the {{domxref("ServiceWorkerRegistration")}} interface returns a reference to the {{domxref('ContentIndex')}} interface, which allows for indexing of offline content.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox">var <var>A ContentIndex object</var> = ServiceWorkerRegistration.index;</pre>
+
+<h3>Value</h3>
+
+<p>A ContentIndex {{jsxref('Object')}}</p>
+
+<h2 id="Examples">Examples</h2>
+
+<p>You can access the property from either your main script or the registered service worker.</p>
+
+<p>Here is an example from the main script:</p>
+
+<pre class="brush: js">
+// reference registration
+const registration = await navigator.serviceWorker.ready;
+
+// feature detection
+if ('index' in registration) {
+
+  // Content Index API functionality
+  const contentIndex = registration.index;
+
+}
+</pre>
+
+<p>From the {{domxref('Service Worker API','service worker')}}:</p>
```

Comment:
```suggestion
<p>From the {{domxref('Service_Worker_API','service worker')}}:</p>
```

Reply:
Make sure you test this in preview. I'm not sure this will work. I think you might need to use an &lt;a> tag.

Reply:
Haha - yeh that won't work - going through all macros now :)

Reply:
Weirdly this _does_ in fact work - it's like this in the push pages too 🤷🏻‍♀️